sklearn/manifold/mds.py

```diff
@@ -129,18 +131,18 @@ def _smacof_single(similarities, metric=True, n_components=2, init=None,
     return X, stress, it + 1
 
 
-def smacof(similarities, metric=True, n_components=2, init=None, n_init=8,
+def smacof(dissimilarities, metric=True, n_components=2, init=None, n_init=8,
```

Comment:
You can call non keyword arguments as keyword arguments, but maybe I am being too thorough. 
@amueller what's the policy on that?